weaviate/batch/crud_batch.py

```diff
@@ -951,6 +953,10 @@ class NonExistingClass not present"
             If weaviate reports a none OK status.
         """
 
+        if self._batch_size is not None and self.dynamic:
+            _Warnings.manual_batching_when_dynamic_enabled()
+            return []
```

Comment:
I think that this line is a breaking change. It should be removed to preserve backwards compatibility